_layouts/deep-page.html

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+editable: true
+class:
+---
+
+<main class="content">
+  <div class="container">
+    {%- include "breadcrumb-nav.html" -%}
```

Comment:
```suggestion
    {%- include 'breadcrumb-nav.html' -%}
```

we will add this component next; will try to keep it simple